lib/ansible/plugins/callback/default.py

```diff
@@ -16,8 +16,25 @@
       - default_callback
     requirements:
       - set as stdout in configuration
+    options:
+      check_mode_markers:
+        name: Show markers when running on check mode
+        description:
+        - "Toggle to control displaying markers when running on check mode. The markers are 'DRY RUN'
+        at the beggining and ending of playbooks execution (when calling ansible-playbook --check)
+        and 'CHECK MODE' as a suffix at every task that is run in 'check' mode."
+        type: bool
+        default: yes
```

Comment:
default should be 'no' to keep current behaviout

also missing `version_added: "2.9"`

Reply:
Default changed to no as per our discussions.
To be enabled after a couple of majors (after 2.9)